data/mitali.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "mitali paliwal",
+  "type": "personal",
+  "bio": "open source community"
+  "links": [
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/_mitali.__/
```

Comment:
```suggestion
  "bio": "open source community",
	"links": [{
		"name": "Follow me on Instagram",
		"url": "https://www.instagram.com/_mitali.__/",
```

Reply:
Hey @mitalipaliwal , make sure your mitali.json file must be in the data folder and your json format is correct.
for json format validation you can use this : https://jsonlint.com/

Reply:
> Hey @mitalipaliwal , make sure your mitali.json file must be in the data folder and your json format is correct.
> for json format validation you can use this : https://jsonlint.com/

@harshit-paneri thank you for reviewing, but in future please avoid duplicating suggestions which have already been made - as this can cause confusion for the author and also additional notifications for the maintainers.